Packs/CortexResponseAndRemediation/Playbooks/playbook-Suspicious_execution_from_tmp_folder.yml

```diff
@@ -0,0 +1,1394 @@
+      id: 1a8ad8be-1a27-4f30-895b-0244fec589db
+      version: -1
+      name: File quarantine
+      description: 'Quarantines a file on selected endpoints. '
+      script: '|||core-quarantine-files'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "56"
+      '#none#':
+      - "57"
+    scriptarguments:
+      endpoint_id_list:
+        simple: ${alert.agentid}
+      file_hash:
+        simple: ${MaliciousFileHash}
+      file_path:
+        simple: ${MaliciousFilePath}
+      interval_in_seconds:
+        simple: "20"
+      timeout_in_seconds:
+        simple: "120"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": -2.5,
+          "y": 2745
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "56":
+    id: "56"
+    taskid: 11c35696-da82-4f76-87a8-3c322b817596
+    type: regular
+    task:
+      id: 11c35696-da82-4f76-87a8-3c322b817596
+      version: -1
+      name: Manual action needed – Malicious file couldn't be quarantined
+      description: |-
+        Dear Analyst,
+
+        The playbook was unable to quarantine the Malicious file due to the following possible reasons:
+
+        - The file is not located on the local host.
+        - The endpoint is currently disconnected.
+        - The hash calculation was unsuccessful.
+
+        Please take manual action to terminate the causality process if needed and quarantine the file.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "57"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -250,
+          "y": 2930
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "57":
+    id: "57"
+    taskid: 5ff8887c-01ef-4da1-8a04-4844f342e8d9
+    type: title
+    task:
+      id: 5ff8887c-01ef-4da1-8a04-4844f342e8d9
+      version: -1
+      name: Quarantine file  - Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "66"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 377.5,
+          "y": 3110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "62":
+    id: "62"
+    taskid: c71ded9a-e5ed-4dd8-8c01-8bd81c82d4b7
+    type: regular
+    task:
+      id: c71ded9a-e5ed-4dd8-8c01-8bd81c82d4b7
+      version: -1
+      name: Create Indicators
```

Comment:
I think we should better use the entity enrichment subplaybook for the enrichment.